presto-mongodb/pom.xml

```diff
@@ -15,14 +15,19 @@
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
         <mongo-java.version>3.6.0</mongo-java.version>
         <mongo-server.version>1.5.0</mongo-server.version>
-        <netty.version>4.0.32.Final</netty.version>
     </properties>
 
     <dependencies>
         <dependency>
             <groupId>org.mongodb</groupId>
             <artifactId>mongo-java-driver</artifactId>
             <version>${mongo-java.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>io.netty</groupId>
+                    <artifactId>*</artifactId>
```

Comment:
My understanding is we usually list these artifacts explicitly. Is there too many artifacts?

Reply:
About 20+ of them